tokio/src/sync/mutex.rs

```diff
@@ -378,8 +378,18 @@ impl<T: ?Sized> Mutex<T> {
     ///
     /// static LOCK: Mutex<i32> = Mutex::const_new(5);
     /// ```
-    #[cfg(all(feature = "parking_lot", not(all(loom, test)),))]
-    #[cfg_attr(docsrs, doc(cfg(feature = "parking_lot")))]
+    ///
+    /// NOTE that this function is available if your rust compiler
+    /// is newer than 1.63 and does not specify `--cfg tokio_no_const_mutex_new`
+    /// or if you enabled feature `parking_lot`.
+    #[cfg(all(
+        any(feature = "parking_lot", not(tokio_no_const_mutex_new)),
+        not(all(loom, test))
+    ))]
+    #[cfg_attr(
+        docsrs,
+        doc(cfg(any(feature = "parking_lot", not(tokio_no_const_mutex_new))))
```

Comment:
tokio_no_const_mutex_new is not a stable public API, and the user likely to actually use it are only our CI and old nightly users. So I do not want it to appear in the docs.

Reply:
Thanks, I've removed this from doc.